android/buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -14,10 +14,10 @@ object Versions {
     const val billingClient = "6.0.1"
 
     object Android {
-        const val compileSdkVersion = 33
+        const val compileSdkVersion = 34
```

Comment:
This should probably be its own PR

Reply:
Agree: https://github.com/mullvad/mullvadvpn-app/pull/5473